src/wallet/rpcwallet.cpp

```diff
@@ -4837,6 +4839,359 @@ UniValue walletcreatefundedpsbt(const JSONRPCRequest& request)
+        throw JSONRPCError(RPC_WALLET_ERROR, strError);
+    }
+    CValidationState state;
+    mapValue_t mapValue;
+    mapValue["comment"] = colorId.toHexString();
+    if (!pwallet->CommitTransaction(tx, std::move(mapValue), {} /* orderForm */, {} /* std::move(fromAccount)*/, reservekey, g_connman.get(), state)) {
+        strError = strprintf("Error: The transaction was rejected! Reason given: %s", FormatStateMessage(state));
+        throw JSONRPCError(RPC_WALLET_ERROR, strError);
+    }
+
+    UniValue result(UniValue::VOBJ);
+    result.pushKV("color", colorId.toHexString());
+    result.pushKV("txid", tx->GetHashMalFix().GetHex());
+    return result;
+}
+
+static UniValue reissuetoken(const JSONRPCRequest& request)
+{
+    std::shared_ptr<CWallet> const wallet = GetWalletForJSONRPCRequest(request);
+    CWallet* const pwallet = wallet.get();
+
+    if (!EnsureWalletIsAvailable(pwallet, request.fHelp)) {
+        return NullUniValue;
+    }
+
+    if (request.fHelp || request.params.size() != 2)
+        throw std::runtime_error(
+            "reissuetoken \"color\" value \n"
+            "\nReissue colored coins or tokens with the given color. The token must be of type REISSUABLE for this RPC to issue the token.\n"
+            + HelpRequiringPassphrase(pwallet) +
+            "\nArguments:\n"
+            "1. \"color\"            (string, required) The tapyrus color / token to be reissued.\n"
+            "2. \"value\"            (numeric or string, required) The amount to issue. eg 0.1\n"
+            "\nResult:\n"
+            "\"txid\"                  (string) The transaction id.\n"
+            "\nExamples:\n"
+            + HelpExampleCli("reissuetoken", "\"c18282263212c609d9ea2a6e3e172de238d8c39cabd5ac1ca10646e23f\" 0.1")
+        );
+
+    const std::vector<unsigned char> vColorId(ParseHex(request.params[0].get_str()));
+    ColorIdentifier colorId(vColorId);
+
+    if(colorId.type != TokenTypes::REISSUABLE)
+        throw JSONRPCError(RPC_INVALID_PARAMETER, "Unknown token type given.");
+
+    throw JSONRPCError(RPC_METHOD_NOT_FOUND, "Not implemented.");
+}
+
+static UniValue transfertoken(const JSONRPCRequest& request)
+{
+    std::shared_ptr<CWallet> const wallet = GetWalletForJSONRPCRequest(request);
+    CWallet* const pwallet = wallet.get();
+
+    if (!EnsureWalletIsAvailable(pwallet, request.fHelp)) {
+        return NullUniValue;
+    }
+
+    if (request.fHelp || request.params.size() != 2)
+        throw std::runtime_error(
+            "transfertoken \"address\" amount \n"
+            "\nSend colored coins or tokens to a given address.\n"
+            + HelpRequiringPassphrase(pwallet) +
+            "\nArguments:\n"
+            "1. \"address\"            (string, required) The colored tapyrus address to send to.\n"
+            "2. \"amount\"             (numeric or string, required) The amount in to send. eg 0.1\n"
```

Comment:
`amount` must be numeric. Token does not allow for unit conversions such as BTC<->Satoshi.